README.md

```diff
@@ -1,14 +1,17 @@
 # Getting Started with the Pub/Sub API
 
-- [About](#about)
-- [Resources](#resources)
+- [About Pub/Sub API](#about-pubsub-api)
+- [gRPC](#grpc)
+- [Documentation and Blog Posts](#documentation-and-blog-posts)
```

Comment:
Sorry, I missed this one. It looks like this should be `[Documentation and Blog Post](#documentation-and-blog-post)`

Reply:
Good catch! Fixed.